css/style.css

```diff
@@ -1,19 +1,352 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/* Variables */
+:root {
+  --color-brand: #e0633a;
+  --color-brand-dark: #e04815;
+  --color-light: #ffffff;
+  --color-darker: #151515;
+  --color-dark: #3c3c3c;
+  --color-mid: #8d8d8d;
+  --color-cr: #dbdbdb;
+
+  --fs-h1: 4rem;
+  --fs-hero-description: 2.5rem;
+  --fs-h2: 3.5rem;
+  --fs-h3: 1.25rem;
+  --fs-navbar: 1.25rem;
+  --fs-p: 2rem;
+  --fs-obs: 1rem;
```

Comment:
I liked that you used variables for font sizes! It really makes it easier for future changes.

Reply:
Thanks. It is especially useful for creating responsive design.